resources/skins/nudus/metadata.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<skin version="0.01">
+  <author>
+    <name>anonymous</name>
```

Comment:
feel free to at least write your nickname here?

Reply:
But what would you suggest to do with the line below?

Reply:
you can leave it empty, that email is redundant really, will probably remove it

Reply:
I cannot, as the theme is not displayed when there is no value. 
Although I didn't try to build it, so I am not certain.

Reply:
db76984b9

Reply:
Thank you.